### PR TITLE
IGMP snooping tests without traffic

### PR DESCRIPTION
**Pull request by AndriyLozovyyPLV**
Added IGMP snooping test cases without traffic:
- test_igmp_snooping[2-2]
- test_igmp_snooping[3-3]
- test_igmp_snooping[2-1]
- test_igmp_snooping[3-1]
- test_igmp_snooping[3-2]
- test_igmp_snooping_modified_query
- test_igmp_snooping_diff_source_addrs

### CODE REVIEW

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    11. Verify MDB entry for leave port is deleted and no traffic is received
+    """
+
+    bridge = "br0"
+    sleep_value = 8
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
+        pytest.skip("The testbed does not have enough dent with tgen connections")
+    dent_dev = dent_devices[0]
+    dev_name = dent_dev.host_name
```
Comment (SerhiyBoikoPLV) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py]:
```suggestion
    dev_name = dent_devices[0].host_name
```

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+    if querier_interval:
+        out = await IpLink.set(
+            input_data=[{device_name: [
+                {"device": bridge, "type": "bridge", "mcast_querier": 1}]}])
+        err_msg = f"Verify that bridge set mcast_querier 1.\n{out}"
+        assert out[0][device_name]["rc"] == 0, err_msg
+
+        out = await IpLink.set(
+            input_data=[{device_name: [
+                {"device": bridge, "type": "bridge", "mcast_querier_interval": querier_interval * 100}]}])
```
Comment (SerhiyBoikoPLV) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py]: you can combine this with the previous command

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+
+
+async def get_bridge_mdb(device_name, options="-d -s -j"):
+    """
+    Get parsed bridge Mdb entries
+    Args:
+        device_name (str): Dent device host_name
+        options (str): Options to pass to bridge cmd
+    Returns:
+        Dictinary with parsed mdb and router entries
```
Comment (yuriilisovskyi) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py]: Dictionary*

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+    return mdb_entires, router_entires
+
+
+async def common_bridge_and_igmp_setup(device_name, bridge, igmp_ver, dut_ports, querier_interval=0):
+    """
+    Common setup part for bridge and igmp
+    Args:
+        device_name (str): Dent device host name
+        bridge (str): Bridge device name
+        igm_ver (int): IGMP Version to set on bridge dev
```
Comment (yuriilisovskyi) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py]: igmp_ver

**Review by yuriilisovskyi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**